### ISSUE TITLE
Allow adminstrative rejection of candidates

### ISSUE THREAD

**Issue by alvestrand** [w3c/webrtc-pc#2534]
In conjunction with the portscanning issue, it was stated (https://github.com/w3c/webrtc-pc/issues/2426#issuecomment-570528505) that "I think we should state that ICE-TCP ports should not connect to ports on the Fetch "bad ports" list - https://fetch.spec.whatwg.org/#port-blocking"

There might be other reasons for administratively rejecting a candidate - for instance overuse of resources, or local policy.

This seems to deserve its own issue.

**Comment by alvestrand** [comment]
Suggested fix: Add to the "in parallel" steps on https://w3c.github.io/webrtc-pc/#dom-peerconnection-addicecandidate:

"If the candidate is disallowed by policy, reject the promise with a newly created InvalidAccessError"

For canddiates present in a remote description, rejecting a description based on a single bad candidate seems excessive. I suggest adding to https://w3c.github.io/webrtc-pc/#set-description, as step 4 under "apply as JSEP with these restrictions":

"Any candidates in the remote description that are disallowed by policy MUST be ignored, and MUST be removed from the SDP before it is set as PendingRemoteDescription."

**Comment by alvestrand** [comment]
In order to make this testable, I suggest that we add (but I'm not sure):
A TCP candidate on the Fetch block list (ref) MUST be rejected.
For testing, an UDP candidate on localhost using the discard port (9) MUST be accepted.

**Comment by alvestrand** [comment]
June WG VI landed on recommending that forbidden candidates be rendered invisible - that they should appear identical to all other candidates for which you get no answer.
This has some effect on stats, and can't be tested with current interfaces.
